AuthenticationExample/AuthenticationExample.xcodeproj/project.pbxproj

```diff
@@ -239,7 +239,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
```

Comment:
Don't forget to update the repo README [here](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit?tab=readme-ov-file#requirements) too!

Reply:
Got it, thanks!